app/controllers/finance/banding_tracker/provider_choices_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Finance
+  module BandingTracker
+    class ProviderChoicesController < BaseController
+      def new
+        @providers = LeadProvider.all
```

Comment:
```suggestion
        @lead_providers = LeadProvider.all
```

We should try and be consistent with the naming.